meldingen_core/actions/attachment.py

```diff
@@ -124,12 +124,22 @@ async def __call__(
 
 
 class ListAttachmentsAction(Generic[A, M]):
+    _attachment_repository: BaseAttachmentRepository[A]
```

Comment:
Het type argument `M` hierboven is in deze situatie ongebruikt. Zou je die weg kunnen halen?

Reply:
Scherp, heb het aangepast!